apicast-gateway/apicast.yml

```diff
@@ -119,6 +128,10 @@ parameters:
   value: production
   name: DEPLOYMENT_ENVIRONMENT
   required: true
+- description: "Number of worker processes to start."
```

Comment:
I think we can remove this

Reply:
done in dedce9bb0089fb28c3993c590908fa07e885c47a